### DISCUSSION TITLE
How to restart a Task when Workflow fails at task 2 in sequence.

### DISCUSSION THREAD

**Discussion by laxmikantb** [Netflix/conductor#2663] [Q&A]
We do requirement like workflow defined as 
    task1-> task2-> decision (T3, t4) -> t5 -> t6 

now workflow get started but failed in task2, is any way we can re-start same workflowid starting from task2 or failed.I do not want restart task1 as task1 already time consuming job and no need to re run task1 , but would like to use output from task1 to restart t2.

**Comment by jxu-nflx** [comment]
Hello @laxmikantb you could use either 'rerun` or `retry` to restart from task2: https://netflix.github.io/conductor/apispec/#manage-workflows
